plugins/generator-1.20.1/forge-1.20.1/templates/livingentity/livingentity.java.ftl

```diff
@@ -824,6 +834,10 @@ public class ${name}Entity extends ${extendsClass} <#if data.ranged>implements R
 		<#if data.spawnInDungeons>
 			DungeonHooks.addDungeonMob(${JavaModName}Entities.${data.getModElement().getRegistryNameUpper()}.get(), 180);
 		</#if>
+
+		<#if data.spawnInRaids>
+		Raid.RaiderType.create("${registryname}", ${JavaModName}Entities.${data.getModElement().getRegistryNameUpper()}.get(), new int[]{0, 4, 3, 3, 4, 4, 4, 2});

```

Comment:
The spawn rates should be something users define (I don't know if this is the place where it's defined, but this is what makes the most sense by looking the code)

Reply:
Yes

Reply:
I think the spawn rates option may be added in the future (because I don't know what each value corresponds to)